src/codeManager.ts

```diff
@@ -297,6 +297,11 @@ export class CodeManager implements vscode.Disposable {
     private getWorkspaceRoot(codeFileDir: string): string {
         return this._workspaceFolder ? this._workspaceFolder : codeFileDir;
     }
+    
+    private getWorkspaceName(workspaceRoot: string): string {

```

Comment:
workspaceRoot --> codeFileDir

Reply:
It is called with `this.getWorkspaceRoot(codeFileDir)`, so I think that `workspaceRoot`is more proper than `codeFileDir`...